### PR TITLE
SMC Major changes

### PR DESCRIPTION
**Pull request by Kitsunemitsu**
## About The Pull Request

Adds two new specialist weapons
Autolathe to the Phobos Mission
Changes Neurotoxin spit from hard knockdown to stamina damage
Adds cap to the Requisitions Chief
Adds Hardsuits to the Andromeda

## Why It's Good For The Game

Make Specialist weapons actually usable as main weapons

## Changelog
:cl:
add: Added Nano-blade
add: Added Minigunpack
add: Added Hardsuits to the Marines
add: Added Flatcap to the Req Chief
del: Removed old things
tweak: Neurotoxin no longer hardknockdowns
/:cl:

### CODE REVIEW

**Comment by Kitsunemitsu** [comment]
Might have had Strong DMM on the wrong settings...

**Comment by EgorDinamit** [comment]
> Major changes

> Files changed: 4

:trollface:

**Comment by Kitsunemitsu** [comment]
It's Major balance and map changes.

I just added new sprites as well

File: ModularBungalow/clothing/smc.dm [hunk 1]
```diff
@@ -21,6 +21,11 @@
 	name = "triage technician's jumpsuit"
 	icon_state = "triagetech"
 
+/obj/item/clothing/under/smc/triageofficer
+	desc = "A medic's sundicate jumpsuit with a hint of green. Helps stand out in the heat of battle."
```
Comment (BootlegBow) [inline ModularBungalow/clothing/smc.dm:25]: woah new 'sundicate' faction confirmed?????? you can't leak our secret ideas in the second line of the pr kirie!!!!!!!1!111111111!

**Review by BootlegBow — APPROVED** [review]
how is it my fault that you hit the wrong key

**Pull request merged**